macros/utils/cross_db_utils/edr_day_of_week_expression.sql

```diff
@@ -0,0 +1,34 @@
+{% macro edr_day_of_week_expression(date_expr) %}
```

Comment:
This works, but it's a mess and for no reason. 
You can cleanup all the `set` and `returns`, like in all the other croos_db_utils.

Reply:
Cleaned up